internal/generator/vector/output/elasticsearch/elasticsearch.go

```diff
@@ -43,12 +43,12 @@ id_key = "_id"
 
 func Conf(o logging.OutputSpec, inputs []string, secret *corev1.Secret, op Options) []Element {
 	outputs := []Element{}
-
+	id_key := "elasticsearch_preprocess_" + strings.ToLower(vectorhelpers.Replacer.Replace(o.Name))
```

Comment:
this generates multiple copies of `elasticsearch_preprocess`

Reply:
This has been fixed.